src/lib.rs

```diff
@@ -1191,11 +1184,13 @@ pub fn new(bytes:[u8;16]) -> Uuid {
 /// # ;
 /// # }
 /// ```
+#[cfg(feature = "uuid_macro")]
+#[macro_export]
 macro_rules! uuid {
```

Comment:
I think this can now be moved to the `uuid_macro` crate and just reexported in `uuid`

Reply:
That‘s good idea,I will try later,because proc_macro_expr_impl must be implemented external crate,If I move it to uuid_macro , the crate uuid_macro may be separated into two crates.